recipe_scrapers/lekkerensimpel.py

```diff
@@ -29,10 +32,39 @@ def image(self):
         return image["content"] if image else None
 
     def ingredients(self):
-        return self.schema.ingredients()
+        if self.schema.ingredients():
+            return self.schema.ingredients()
+
+        ingredient_header = self.soup.find("strong", string="Benodigdheden:")
+        if ingredient_header:
+            return self.process_ingredients(
+                ingredient_header.parent.parent.find_next("ul")
+            )
+
+        ingredient_header = self.soup.find("p", string="Benodigdheden:")
+        if ingredient_header:
+            return self.process_ingredients(ingredient_header.parent.find_next("ul"))
+
+        raise ElementNotFoundInHtml("Could not find ingredients.")
+
+    def process_ingredients(self, container):
+        ingredients = container.findChildren("li")
+
+        return [normalize_string(i.get_text()) for i in ingredients]
 
     def instructions(self):
-        return self.schema.instructions()
+        if self.schema.instructions():
+            return self.schema.instructions()
+
+        instructions_head = self.soup.find("strong", string="Bereidingswijze:")
+        if instructions_head:
+            return instructions_head.parent.find_next("p")
+
+        instructions_head = self.soup.find(string=re.compile("Bereidingswijze"))
```

Comment:
The problem here is the html is like this:

```
<strong>Bereidingswijze:<br/>
</strong>
```
I think this is even invalid html, but the `<br/>` in the paragraph did block me from using a normal find by tag. Maybe you know how to solve this?

Reply:
Ok, yep.. I didn't know about that behaviour.  It sounds like https://bugs.launchpad.net/beautifulsoup/+bug/1713129 - the `.string` property for the `strong` becomes `None` because it's ambiguous.  There's a feature request to implement different behaviour.

Knowing that, I'm OK with the current approach.  I'll spend some time thinking about alternative ideas, but it may be the best we can do at the moment.